ballerinaByExample/tools/generate.go

```diff
@@ -457,10 +457,10 @@ func  parseExamples(categories []BBECategory) []*Example {
             metatagsFilePath := fileDirPath + exampleBaseFilePattern + metatagsFileExtn
 
             balFiles := getAllBalFiles(fileDirPath);
-            if len(balFiles) == 0 {
-                fmt.Fprintln(os.Stderr, "\t[WARN] Skipping bbe : " + exampleName + ". No *.bal files are found")
-                continue
-            }
+            //if len(balFiles) == 0 {
+                //fmt.Fprintln(os.Stderr, "\t[WARN] Skipping bbe : " + exampleName + ". No ///*.bal files are found")
+                //continue
+            //}
```

Comment:
Shall we remove this rather than commenting?

Reply:
We might need the logic again later.

Reply:
Ack